interface/forms/procedure_order/new.php

```diff
@@ -403,13 +423,13 @@ function validate(f) {
 ?>
 </p>
 
-<center>
+
 
 <p>
-<table border='1' width='95%' id='proctable'>
+<table  width='100%' id='proctable' class="table">
```

Comment:
Is this class name needed, given that the table can already be identified in css by its id?